cumulusci/tasks/release_notes/generator.py

```diff
@@ -134,8 +134,9 @@ def aggregate_child_change_notes(self, pull_request):
         Child pull requests are pull requests that have a base branch
         equal to the the given pull request's head."""
         self.change_notes = get_pull_requests_with_base_branch(
-            self.repo, pull_request.head.label.split(":")[1]
+            self.repo, pull_request.head.ref
         )
+        self.change_notes = list(filter(is_merged, self.change_notes))
```

Comment:
Consider

    self.change_notes = [note for note in self.change_notes if is_merged(note)]

but I don't feel strongly about it

Reply:
Yours is definitely more readable